### PR TITLE
New manipulation functions: filter & putInState

### PR DESCRIPTION
**Pull request by Dawidpol**

### CODE REVIEW

File: src/manipulation.ts [hunk 1]
```diff
@@ -8,4 +10,27 @@ export const mapPropertyDown = (source: string, target: string, defaultValue?: a
 export const mapPropertyUp = (source: string, target: string, defaultValue?: any) => async (ctx, next?) => {
   next && await next()
   ctx.state[target] = get(ctx.state, source, defaultValue || null)
+}
+
+export const putInState = async (variableName: string) => {
```
Comment (matiboy) [inline src/manipulation.ts]: Since it's from event, could we call it `eventToState` ? Not obvious, cause it could be from context

**Review by matiboy — COMMENTED** [review]
just a change of naming please

**Comment by yujinlim** [comment]
@Dawidpol @matiboy i don't think there should be any release version update on a PR, this should be only implemented either on develop branch(pre-release) /master branch (release).

**Comment by matiboy** [comment]
@yujinlim Agree.
@Dawidpol Please note, thanks

**Pull request merged**